packages/core/src/Stores/client-store.js

```diff
@@ -778,6 +779,10 @@ export default class ClientStore extends BaseStore {
         return this.residence_list.find(item => item.value === this.residence)?.disabled === 'DISABLED';
     }
 
+    getResidence() {
+        return this.residence;
+    }
```

Comment:
We already have residence in client store. 
```
 get residence() {
        if (this.is_logged_in) {
            return this.account_settings?.country_code ?? '';
        }
        return '';
    }
```

why do we need this new getResidence?

Reply:
Sorry! I missed this. Will use this.
Thanks